src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -59,13 +65,21 @@ public class Ledger : ILedger, IDisposable
         public Cid LatestKnownDelta { get; private set; }
         public bool IsSynchonising => Monitor.IsEntered(_synchronisationLock);
 
-        public Ledger(IAccountRepository accounts,
+        public Ledger(IKvm virtualMachine,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constructor has 9 parameters, which is greater than the 7 authorized.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)